helix-view/src/document.rs

```diff
@@ -185,6 +186,8 @@ pub struct Document {
 
     // when document was used for most-recent-used buffer picker
     pub focused_at: std::time::Instant,
+
+    pub last_view: ViewPosition,
```

Comment:
Minor nit but let's call this `last_view_position` since it's easy to confuse this as being the last View that focused the doc